fusequery/query/src/pipelines/transforms/transform_filter.rs

```diff
@@ -93,6 +94,14 @@ impl Processor for FilterTransform {
             // Downcast to boolean array
             let filter_array = filter_array.bool()?.downcast_ref();
             // Convert to arrow record_batch
+
+            let mut  filter_exit_true =  filter_array.values().chunks::<u64>();
+
+            if  filter_exit_true.find( |p| p.count_ones() as usize > 0_usize ).is_none() &&
```

Comment:
Good point, can use just use `!filter_exit_true.iter().any(|v| *v > 0) && ...`

Reply:
I've   pprof. It seems that chunks is better